docs/hopsml/index.rst

```diff
@@ -15,4 +15,5 @@ HopsML
    maggy.rst
    sklearn_model_serving.rst   
    tf_model_serving.rst
+   provenance.rst
```

Comment:
Can we also put it under the user guide menu?

Reply:
done